Biblio.bib

```diff
@@ -171,6 +186,7 @@ @Article{szabo1997formalizing
 }
 
 @InProceedings{miller1997future,
+	url = {{https://drive.google.com/file/d/0Bw0VXJKBgYPMS0J2VGIyWWlocms/edit?usp=sharing}},
```

Comment:
Who uploaded this video?  Did the person in the video agree?  Did the person who created the video agree?

Reply:
Yes if you go the sharing properties you will see that Miller himself is sharing it, which I mentioned before.

Reply:
I didn't know these three dots.  Thanks.